{{cookiecutter.project_slug}}/clients/vue3/package.json

```diff
@@ -13,15 +13,17 @@
     "heroku-postbuild": "vue-cli-service build"
   },
   "dependencies": {
+    "@headlessui/vue": "^1.7.7",
+    "@heroicons/vue": "^2.0.14",
     "@thinknimble/tn-forms": "^2.0.9",
     "@thinknimble/tn-models": "1.2.1",
     "@thinknimble/tn-utils": "1.6.0",
-    "autoprefixer": "^9.8.8",
+    "autoprefixer": "^10.0.2",
     "axios": "^0.22.0",
     "core-js": "^3.6.5",
     "js-cookie": "^3.0.1",
-    "postcss": "^7.0.39",
-    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.17",
```

Comment:
**Note:** When I was first testing the Tailwind V3 installation, I had to _uninstall_ the `autoprefixer`, `postcss`, and `tailwindcss` packages individuall and then re-install all of them individually (w/ the associated package versions listed here).

This is mostly just an FYI for folks who want to upgrade an existing project using Tailwind V2 to be using Tailwind V3.

Reply:
This was a godsend. Thank you for documenting this!